compio-quic/src/connection.rs

```diff
@@ -210,13 +209,14 @@ impl ConnectionInner {
                     }
                     state
                 },
-                BufResult::<(), Vec<u8>>(res, mut buf) = transmit_fut => match res {
-                    Ok(()) => {
-                        buf.clear();
-                        send_buf = Some(buf);
-                        self.state()
-                    },
-                    Err(e) => break Err(e),
+                BufResult::<(), Vec<u8>>(res, mut buf) = transmit_fut => {
+                    #[allow(unused)]
+                    if let Err(e) = res {
+                        error!("I/O error: {}", e);
```

Comment:
So the error is only occasional? We don't need to do anything to recover?

Reply:
[QUIC's loss recovery mechanism](https://datatracker.ietf.org/doc/html/rfc9002#loss-detection) should be sufficient to handle it. In fact, as seen in the logs, `quinn-proto` will immediately notice the loss and take action.

Reply:
Fair enough